src/layouts/components/TextInput.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
```

Comment:
can move this to src/components/commons

Reply:
refactored, will be keeping it in the current layout since its not a common component anymore